pkg/controlplane/store/types.go

```diff
@@ -101,3 +103,23 @@ func NewBinding(binding *api.Binding) *Binding {
 		Version:     bindingStructVersion,
 	}
 }
+
+// AccessPolicy to allow/deny specific connections
+type AccessPolicy struct {
+	api.Policy
+	// Version of the struct when object was created.
+	Version uint32
+}
+
+// Keys return the keys identifying the policy.
+func (ap *AccessPolicy) Keys() []string {
+	return []string{ap.Name}
+}
+
```

Comment:
```suggestion
```

This is actually a leftover from the time where a generic store existed.

Reply:
Removed. Should I also remove the other implementations for the `Keys()` method?
```golang
func (exp *Export) Keys() []string
func (imp *Import) Keys() []string
func (b *Binding) Keys() []string
```

Reply:
I've seen you already fixed bugs you found on the way (thanks!).
You can fix that as well, if you like.